core/unix/signal_private.h

```diff
@@ -214,7 +214,11 @@ typedef _STRUCT_UCONTEXT /* == __darwin_ucontext */ kernel_ucontext_t;
  * (these are from /usr/src/linux/arch/i386/kernel/signal.c for kernel 2.4.17)
  */
 
-#    define RETCODE_SIZE 8
+#    if defined(X86)
+#        define RETCODE_SIZE 8
+#    elif defined(ARM)
+#        define RETCODE_SIZE 16
```

Comment:
Hmm, could you explain where this came from?

Reply:
It's from https://elixir.bootlin.com/linux/latest/source/arch/arm/kernel/signal.h#L5

Reply:
I see, thanks.